lib/unknown-attributes-filter.coffee

```diff
@@ -0,0 +1,43 @@
+debug = require('debug')('sphere-product-import:unknown-attributes-filter')
+_ = require 'underscore'
+_.mixin require 'underscore-mixins'
+Promise = require 'bluebird'
+
+class UnknownAttributesFilter
+
+  constructor: (@logger) ->
+    debug "Unknown Attributes Filter initialized."
```

Comment:
Sorry, just got this is the class name - the name is extremely misleading esp. when the words are space separated here.